src/Widgets/DataLabelWidget/DataLabelWidgetComponent.tsx

```diff
@@ -50,34 +51,37 @@ provideComponent(DataLabelWidget, ({ widget }) => {
   )
 })
 
-const AttributeValue = connect(function AttributeValue({
-  datetimeFormat,
-  showAs,
-}: {
-  datetimeFormat: string | null
-  showAs: string | null
-}) {
-  const rawAttributeValue = useData().dataItemAttribute()?.get()
-
-  const enumOptions = useEnumOptions()
-  const matchingOption = enumOptions.find(
-    ({ value }) => value === rawAttributeValue,
-  )
-
-  const attributeValue = matchingOption?.title ?? rawAttributeValue
-
-  if (showAs === 'currency') return <Currency value={attributeValue} />
-  if (showAs === 'datetime') {
-    return <Datetime value={attributeValue} datetimeFormat={datetimeFormat} />
-  }
-  if (showAs === 'link') return <Link value={attributeValue} />
-
-  if (typeof attributeValue === 'number') {
-    return <NumberText value={attributeValue} />
-  }
-
-  return <Text value={attributeValue} />
-})
+const AttributeValue = connect(
+  function AttributeValue({
+    datetimeFormat,
+    showAs,
+  }: {
+    datetimeFormat: string | null
+    showAs: string | null
+  }) {
+    const rawAttributeValue = useData().dataItemAttribute()?.get()
+
+    const enumOptions = useEnumOptions()
+    const matchingOption = enumOptions.find(
+      ({ value }) => value === rawAttributeValue,
+    )
+
+    const attributeValue = matchingOption?.title ?? rawAttributeValue
+
+    if (showAs === 'currency') return <Currency value={attributeValue} />
+    if (showAs === 'datetime') {
+      return <Datetime value={attributeValue} datetimeFormat={datetimeFormat} />
+    }
+    if (showAs === 'link') return <Link value={attributeValue} />
+
+    if (typeof attributeValue === 'number') {
+      return <NumberText value={attributeValue} />
+    }
+
+    return <Text value={attributeValue} />
+  },
+  { loading: Loading },
```

Comment:
* [ ] FYI Typically many labels are used on the same details page. Using the regular Loading spinner for each data label introduces a lot of animation in many places. A visitor may be confused by seeing more than one spinner and the page may look broken. I suggest to use a more decent placeholder instead of a spinner in these places. Quick prototype: `{ loading: () => <span className="opacity-25">▆▆▆▆</span> }`


<img width="48%" alt="image" src="https://github.com/user-attachments/assets/33945b1e-ad56-4d8d-8fb2-585a19e9b5da">
<img width="48%" alt="image" src="https://github.com/user-attachments/assets/b8e9f440-7a3a-4503-b4f2-b09895b3a2e2">

Reply:
To me this is good enough for now. We can improve later on, if there is demand.